api/tests/unit/infrastructure/adapters/assessment-adapter_test.js

```diff
@@ -129,5 +134,124 @@ describe('Unit | Adapter | Assessment', () => {
       });
     });
   });
+  describe('#getSmartAssessment', () => {
+
+    it('should return an Assessment from the Smart repository', () => {
```

Comment:
C'est quoi le Smart repository ?

Reply:
Rien du tout, erreur dans l'écriture d'un message "should return an Assessment of SmartRandom Algo"